lib/improver/tests/utilities/test_cube_checker.py

```diff
@@ -216,5 +221,54 @@ def test_two_way_mismatch(self):
         self.assertListEqual(result, ["time", "realization"])
 
 
+class Test_find_percentile_coordinate(IrisTest):
+
+    """Test whether the cube has a percentile coordinate."""
+
+    def setUp(self):
+        """Create a wind-speed and wind-gust cube with percentile coord."""
+        data = np.zeros((2, 2, 2, 2))
+        self.wg_perc = 50.0
+        self.ws_perc = 95.0
+        gust = "wind_speed_of_gust"
+        self.cube_wg = (
+            create_cube_with_percentile_coord(data=data,
+                                              perc_values=[self.wg_perc, 90.0],
+                                              standard_name=gust))
+
+    def test_basic(self):
```

Comment:
Just noticed you don't seem to have a test that checks that the correct coordinate is returned, this test only checks the type.